### PR TITLE
manifest: update npmx to have fixed DIE temperature calculations

### PR DESCRIPTION
**Pull request by wilu-nordic**

### CODE REVIEW

**Comment by nordic-segl** [comment]
PR to test repo, that enables affected test is here:
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-sdk-npmx/pull-requests/155/diff#conf/ci/pipeline_utils.groovy

Integration CI is passing (build #1 in on develop, #2 is on this PR):
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_integration/view/change-requests/job/PR-155/

Waiting for results of nightly tests (builds #1-4 are on develop, #5 is on this PR):
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_develop_samples/view/change-requests/job/PR-155/

**Review by nordic-segl — DISMISSED** [review]

**Review by magp-nordic — APPROVED** [review]

**Review by nordic-segl — DISMISSED** [review]

**Review by nordic-segl — APPROVED** [review]
Integration CI is passing (#4):
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_integration/view/change-requests/job/PR-155/
Nightly scope is passing (#16):
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_develop_samples/view/change-requests/job/PR-155/

**Comment by nordic-segl** [comment]
Integration Ci will fail until test repo is aligned to changes from this PR
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-sdk-npmx/pull-requests/155/diff#bts/cli/npmx_cli_base.py

**Comment by masz-nordic** [comment]
@wilu-nordic can you rebase?
@nordic-segl can you rerun tests after that rebase?

**Comment by nordic-segl** [comment]
waiting for results of integration CI aligned to this PR (build #5):
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_integration/view/change-requests/job/PR-155/

**Review by nordic-segl — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ manifest:
   projects:
     - name: npmx
       path: modules/npmx
-      revision: 0ac6e7806e19e89af0d5dd909dd4ca353cd9f2bd
+      revision: pull/123/head
```
Comment (nordic-segl) [inline west.yml:24]: 97bde816d620f32ea26c62319a3553b133540ee6
https://github.com/nrfconnect/npmx/commits/main

File: drivers/npmx/npmx_shell.c [hunk 2]
```diff
@@ -619,10 +619,12 @@ static int cmd_die_temp_set(const struct shell *shell, size_t argc, char **argv,
...
 
 	int err = 0;
-	uint16_t temperature = CLAMP(shell_strtoul(argv[1], 0, &err), 0, UINT16_MAX);
+	int16_t temperature = (int16_t)CLAMP(shell_strtol(argv[1], 0, &err),
+					     NPMX_PERIPH_CHARGER_DIE_TEMPERATURE_MIN_VAL,
+					     NPMX_PERIPH_CHARGER_DIE_TEMPERATURE_MAX_VAL);
 
 	if (err != 0) {
-		shell_error(shell, "Error: temperature has to be a non-negative integer.");
+		shell_error(shell, "Error: temperature has to be an integer.");
```
Comment (masz-nordic) [inline drivers/npmx/npmx_shell.c:627]: `Allowed values are between %d and %d`.

Comment (masz-nordic) [reply drivers/npmx/npmx_shell.c:627]: In a separate PR to speed things up.

**Review by nordic-segl — APPROVED** [review]

**Review by masz-nordic — APPROVED** [review]

**Pull request merged**